float64 utmNorthing
float64 utmEasting
string utmZone
